src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/OffenderController.kt

```diff
@@ -98,7 +99,6 @@ class OffenderController(private val offenderService: OffenderService, private v
       ),
     ],
   )
-  @Transactional
```

Comment:
Removed this as our transaction boundaries are at the service layer rather than controller.
(The service layer has the appropriate annotation)